package.json

```diff
@@ -8,13 +8,14 @@
   "files": [
     "dist"
   ],
-  "description": "",
+  "description": "Bridging mechanism for a two-way communication system between a webview & web (js).",
   "main": "./dist/bridget.js",
   "scripts": {
     "release": "standard-version && git push --follow-tags origin master",
     "prebuild": "rimraf dist",
     "build": "npm-run-all --parallel build:*",
-    "build:umd": "webpack --output-filename bridget.js"
+    "build:umd": "webpack --mode=production --output-filename bridget.js",
+    "build-dev": "webpack --mode=development --output-filename bridget.js"
```

Comment:
one with semicolon, one with dash... and a generic one that is not necessary (parallel only for one?!)... a bit unpretty